utils/ccloud_library.sh

```diff
@@ -69,50 +69,49 @@ function ccloud::validate_expect_installed() {
 
   return 0
 }
-function ccloud::validate_ccloud_cli_installed() {
-  if [[ $(type ccloud 2>&1) =~ "not found" ]]; then
-    echo "'ccloud' is not found. Install Confluent Cloud CLI (https://docs.confluent.io/ccloud-cli/current/install.html) and try again"
+function ccloud::validate_cli_installed() {
```

Comment:
@brianstrauch GH isn't allowing me to comment on line 31, but IIUC it should be:

```
CLI_MIN_VERSION=${CLI_MIN_VERSION:-2.2.0}
```